pkg/error/error_test.go

```diff
@@ -315,18 +315,11 @@ func TestFormatErrors(t *testing.T) {
 		},
 	}
 
-	// wannabe test
-	config1 := config.Config{}
-	s.MatchSnapshot(t, FormatErrors(input, config1))
-
-	config2 := config.Config{Format: "gcc"}
-	s.MatchSnapshot(t, FormatErrors(input, config2))
-
-	config3 := config.Config{Format: "github-actions"}
-	s.MatchSnapshot(t, FormatErrors(input, config3))
-
-	config4 := config.Config{Format: "codeclimate"}
-	s.MatchSnapshot(t, FormatErrors(input, config4))
+	for _, format := range outputformat.ValidOutputFormats {
+		t.Run(string(format), func(t *testing.T) {
```

Comment:
Add a fmt.Stringer interface to your OutputFormat type

This way you could use format without casting the type here

It's as simple as this

```go
func (f OutputFormat) String() string {
    return string(f)
}
```

Reply:
I added the interface, but `t.Run` insists on getting a string. It helped in two other spots in but sadly here it doesn't work:
> editorconfig-checker/pkg/error/error_test.go:319:9 cannot use format (variable of type outputformat.OutputFormat) as string value in argument to t.Run (compile)

Reply:
Ob that's right!